chains/evm/listener/event-handler.go

```diff
@@ -54,11 +62,12 @@ func (eh *DepositEventHandler) HandleEvent(startBlock *big.Int, endBlock *big.In
 
 			m, err := eh.depositHandler.HandleDeposit(eh.domainID, d.DestinationDomainID, d.DepositNonce, d.ResourceID, d.Data, d.HandlerResponse)
 			if err != nil {
-				log.Error().Err(err).Str("start block", startBlock.String()).Str("end block", endBlock.String()).Uint8("domainID", eh.domainID).Msgf("%v", err)
+				logger.Error().Err(err).Str("start block", startBlock.String()).Str("end block", endBlock.String()).Uint8("domainID", eh.domainID).Msgf("%v", err)
```

Comment:
If we are going to use traces we don't to double them with logs.
We log an error anyway in the listener so this just doubles that unnecessarily.

Reply:
Yes of that's the plan, although I do not want to remove all the logs now. Now I want to leave both logs and traces and gradually migrate toward having important information only in Traces.